### ISSUE TITLE
번역어 합의

### ISSUE THREAD

**Issue by Violet-Bora-Lee** [Yeonjj/javascript-tutorial-ko#12]
번역어의 경우 출판된 책을 기준으로 정하였으면 좋겠습니다.
저는 인사이트의 프론트엔드 개발자를 위한 자바스크립트 프로그래밍(니콜라스 자카스, 한선용) 책을 기준으로 작업하도록 하겠습니다. 일단 공부하면서 빈틈 생기는 것들 메꾸려고 제 리파지터리를 따로 파서 작업중인데 이곳에 지속적으로 진행사항을 이슈로 업데이트하도록 하겠습니다.

**Comment by Yeonjj** [comment]
네 알겠습니다. 혹시 나중을 위해서 일단 collaborator 추가 보내드렸습니다. 
책 내용도 한번 살펴보겠습니다. 합의 내용은 따로 정리하는 게 좋을까요?

**Comment by Violet-Bora-Lee** [comment]
네. 나중에 한국어 리파지토리가 만들어지만 합의가 되어있는게 좋을것같아요. 일단 해당 md파일에 나오는 최초 키워드는 영한병기를 하는게 좋을것 같습니다.

**Comment by Yeonjj** [comment]
그럼 일단 말씀하신 데로 한국어 영어를 같이 표기하는 게 좋겠군요.

**Comment by Violet-Bora-Lee** [comment]
현재까지 사용한 번역어는 다음과 같습니다. 구글 계정을 알려주시면 편집권한을 드리도록 하겠습니다.
https://docs.google.com/spreadsheets/d/1mxlxTwqdQwxkxrqMWqHgGT6lCKgj8HqCZ3UioQLAvL0/edit#gid=0
한-영 병기 예시는 다음과 같습니다. `프로퍼티(property)`

**Comment by Yeonjj** [comment]
넵 번역어는 계속 업데이트 해 주시면 다른 분들이 사용한 번역어와 취합하여 이후에 따로 정리해서 여기 저장소나 혹은 다른 방법으로 번역하시는 분들 위해서 공유하도록 하겠습니다.

**Comment by Yeonjj** [comment]
네 감사합니다 제 구글 계정은 petechoi10@gmail.com입니다.

**Comment by ejolie** [comment]
저도 번역에 참여하고자 하는데, 추가해주시면 감사하겠습니다~! ejisvip@gmail.com 입니다.

**Comment by Violet-Bora-Lee** [comment]
**번역어(번역 컨벤션)**

1. 경어체를 사용합니다.

2. 번역어는 출판된 도서, 국립국어원의 외래어 표기법 용례, 한글라이즈 사이트를 기준으로 하겠습니다. 주 참고자료는 다음과 같습니다
  - 프론트엔드 개발자를 위한 자바스크립트 프로그래밍
  - 인사이드 자바스크립트
  - 러닝 자바스크립트
  - 국립국어원 외래어 표기법 용례 찾기
  - 한글라이즈
3. 주제에서 새롭게 등장하는 키워드는 한-영 병기(예: 프로퍼티(property))를 하였습니다.
4. 번역어는 아래 시트에서 확인할 수 있습니다. 해당 시트를 확인하시어 번역어 통일에 힘써주시기 바랍니다. 새롭게 작업을 시작하는 분들은 구글 계정을 알려주시면 편집 권한을 드리겠습니다.
https://docs.google.com/spreadsheets/d/1mxlxTwqdQwxkxrqMWqHgGT6lCKgj8HqCZ3UioQLAvL0/edit#gid=0
5. 번역이 끝나면 부산대 맞춤법 검사기 등을 이용하여 꼭 맞춤법 검사를 해주세요.

https://github.com/Violet-Bora-Lee/javascript-tutorial-ko

**Comment by jungin-jin-choi** [comment]
보라님 안녕하세요 정인이에요 :D 이메일 luvimperfection@gmail.com 입니다
